nncf/quantization/algorithms/bias_correction/backend.py

```diff
@@ -0,0 +1,223 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from abc import ABC
+from abc import abstractmethod
+from typing import List, Tuple, TypeVar
+
+import numpy as np
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.common.graph.transformations.commands import TransformationCommand
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.tensor import NNCFTensor
+from nncf.common.graph import NNCFGraph
+from nncf.common.graph import NNCFNode
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
+from nncf.common.tensor_statistics.collectors import ReductionShape
+from nncf.common.utils.registry import Registry
+from nncf.common.graph.model_transformer import ModelTransformer
+
+TModel = TypeVar('TModel')
+OutputType = TypeVar('OutputType')
+ALGO_BACKENDS = Registry('algo_backends')
+
+
+class BCAlgoBackend(ABC):
```

Comment:
Can you please use `BiasCorrection` everywhere instead of `BC`?

Reply:
Done